src/tests/components/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { ErrorBoundary } from '../../components/UI'
+
+const ErrorText = 'Error message'
+const ErrorChild = <div className='custom-class'>{ErrorText}</div>
+
+describe(`ErrorBoundary component test`, () => {
+  it('should component rendered properly with status [false]', () => {
```

Comment:
Rename: `should render with status [false]`

Reply:
I will change it after merge because there is more file which needs to be changed